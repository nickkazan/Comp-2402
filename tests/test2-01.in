Even1
Odd1
Even2
Odd2
Even3
Odd3
Even4
Odd4
Even5
Odd5
Even6
Odd6
