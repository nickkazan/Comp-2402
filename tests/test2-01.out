Even1
Even2
Even3
Even4
Even5
Even6
Odd1
Odd2
Odd3
Odd4
Odd5
Odd6
